Application/src/view/visuals/component/Message.java

```diff
@@ -293,6 +311,8 @@ else if (switchDirection && !switchImage) {
 
             //Checks if up image is supposed to be shown. if this one is used it is a return message.
             else if(switchImage){
+                // if it is a return message then flip the direction of the trail arrows
+                rotated = true;
```

Comment:
Here as well, i get it through the comment but it is still a bit confusing to me.